AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/ALAppDelegate.swift

```diff
@@ -21,6 +22,14 @@ class ALAppDelegate: UIResponder, UIApplicationDelegate
         ALSdk.shared()!.mediationProvider = ALMediationProviderMAX
         ALSdk.shared()!.initializeSdk(completionHandler: { configuration in
             // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+            
+            // initialize Adjust SDK
+            let adjustConfig = ADJConfig(
+                appToken: "{YourAppToken}",
+                environment: ADJEnvironmentSandbox)
+
+            adjustConfig?.logLevel = ADJLogLevelVerbose
+            Adjust.appDidLaunch(adjustConfig)
```

Comment:
We discussed that setting verbose logging isn't necessary, and now ObjC is inconsistent with Swift:

```suggestion
            // Initialize Adjust SDK
            let adjustConfig = ADJConfig(appToken: "{YourAppToken}", environment: ADJEnvironmentSandbox)
            Adjust.appDidLaunch(adjustConfig)
```